.github/workflows/release.yml

```diff
@@ -21,15 +21,16 @@ jobs:
         uses: pypa/cibuildwheel@v2.16.5
         env:
           CIBW_SKIP: "cp36-* cp37-* pp* *musl*"
-          CIBW_TEST_SKIP: "*-macosx_arm64 *-macosx_universal2:arm64 *-win32 *-manylinux_i686 cp310-manylinux*"
+          CIBW_TEST_SKIP: "*-macosx_arm64 *-macosx_universal2:arm64 *-win32 *-win_amd64 *-manylinux_i686 cp310-manylinux*"
           CIBW_MANYLINUX_X86_64_IMAGE: quay.io/pypa/manylinux2014_x86_64:latest
           CIBW_MANYLINUX_I686_IMAGE: quay.io/pypa/manylinux2014_i686:latest
           CIBW_ARCHS_MACOS: x86_64 universal2
-          CIBW_TEST_REQUIRES: pytest qiskit-aer
+          CIBW_TEST_REQUIRES: pytest qiskit-aer setuptools
           CIBW_TEST_COMMAND_LINUX: pytest -p no:warnings /mthree_test
           CIBW_TEST_COMMAND_WINDOWS: pytest -p no:warnings C:\Users\RUNNER~1\AppData\Local\Temp\mthree_test
           CIBW_TEST_COMMAND_MACOS:  pytest -p no:warnings /tmp/mthree_test
           CIBW_ENVIRONMENT_LINUX: MTHREE_OPENMP=1
+          CIBW_ENVIRONMENT_MACOS: MACOSX_DEPLOYMENT_TARGET=10.15
```

Comment:
This is more restrictive than what Qiskit uses, but I think it's ok for the typical users of mthree to require a newer macOS target.

Reply:
Huh, good point - I had assumed that the macos-10.15 image mthree was using before would have just defaulted to a build target of 10.15, but actually it was using 10.9.  This PR was an unintentional bump - I'd _meant_ to keep the same version.